content/en/docs/reference/access-authn-authz/admission-controllers.md

```diff
@@ -624,16 +624,12 @@ versions 1.9 and later).
 
 Yes.
 
-For Kubernetes version 1.10 and later, we recommend running the following set of admission controllers using the `--enable-admission-plugins` flag (**order doesn't matter**).
+For Kubernetes version 1.10 and later, the recommended admission controllers are enabled by default, so you do not need to explicitly specify them. You can enable additional admission controllers beyond the default set using the `--enable-admission-plugins` flag (**order doesn't matter**).
```

Comment:
nit: - will be good to list the default admission controllers otherwise will confuse folks as there are lots of tools/ K8s distro around and you don't know which controllers been selected.

Reply:
Yes, but the previous documentation did not solve this problem either; the list was out of date.

Reply:
@DanyC97  #14808

Reply:
thanks @rptaylor for following with a PR